src/Alarm.cpp

```diff
@@ -53,31 +53,31 @@ void Alarm::determineBasePhotoresistorReading(){
     _greenLED->flash();
   }
   _baseReading = avgReading / numOfReadings;
-  Logger::Log("Base Photoresistor Reading Determined");
+  Serial.write(24021); //24021 = Data, Info, Base Photoresistor Data
   _laser->on();
 }
 
 void Alarm::alertFailedAction(){
-  Logger::Log("Alarm Failed Action");
+  Serial.write(14017); //14017 = Log, Info, Alarm Failed
   _buzzer->soundNegativeTone();
   _redLED->flash(1000);
 }
 
 void Alarm::alertSuccessfulAction(){
-  Logger::Log("Alarm successful action");
+  Serial.write(14018); //14018 = Log, Info, Alarm Successful
   _buzzer->soundAffirmativeTone();
   _greenLED->flash(1000);
 }
 
 void Alarm::arm(){
   _laser->on();
   if (!this->isReadyToArm()){
-   Logger::Log("Alarm failed to arm");
+   Serial.write(14019); //14011 = Log, Info, Alarm Failed Arming
    this->alertFailedAction();
    if(!this->isArmed()) _laser->off();
  } else {
    this->alertSuccessfulAction();
-    Logger::Log("Alarm sucessfully armed");
+    Serial.write(14018); //14018 = Log, Info, Alarm Successful
```

Comment:
I think that, based on the additions to the Translation Chart, that it would be more appropriate to send a `020`. I think that was the original intention but it was just overlooked.